packages/pdf-generator/src/components/Paragraph.js

```diff
@@ -0,0 +1,45 @@
+import { Text, View } from '@react-pdf/renderer';
+import { customProps, styleProps } from '../utils/propTypes';
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { customTitle } from '../utils/text';
+import styles from '../utils/styles';
+
+const appliedStyles = styles();
+
+const Paragraph = ({
+    style,
+    children,
+    paragraphProps,
+    ...props
+}) => {
+    return (
```

Comment:
ah can simplify this to just return the html block

`Paragraph = ({
    style,
    children,
    paragraphProps,
    ...props
}) => <View`